.github/config/.markdown-lint.yml

```diff
@@ -0,0 +1,4 @@
+MD013:  # Line length
+  line_length: 120
```

Comment:
If you formated the README with a line length of 100 chars, then the limit should also be 100 and not 120.
Otherwise it will look somehow confusing.

Reply:
Good point!
I kept the `120` in the yml file and adjusted the line length of the README to 120. I feel like this is a reasonable width for a README.